Makefile

```diff
@@ -171,7 +171,7 @@ endif
 # be JIT-compiled by the updated driver from the included PTX.
 ifeq ($(USE_CUDA), 1)
 ifeq ($(origin CUDA_ARCH), undefined)
-	KNOWN_CUDA_ARCHS := 30 35 50 52 60 61
+	KNOWN_CUDA_ARCHS := 30 35 50 52 60 61 70
```

Comment:
Yes, the line below that tests passing every architecture in KNOWN_CUDA_ARCHS to nvcc and chooses only the ones that do not give an error.